coldfront/plugins/qumulo/management/commands/add_qumulo_allocation_attribute_type.py

```diff
@@ -126,3 +126,119 @@ def handle(self, *args, **options):
             is_changeable=True,
             is_unique=False,
         )
+
+        # jprew - here is where I'm adding ITDEV-35273-related attribute types
+        # for the allocation collection
+        AllocationAttributeType.objects.get_or_create(
+            attribute_type=AttributeType.objects.get(name="Yes/No"),
+            name="secure",
+            is_required=True,
+            is_private=False,
+            is_changeable=False,
+        )
+
+        # indicates whether the allocation is subject to being audited
```

Comment:
@cdeanj2 some of these fields have already been created. Please reconcile with the latest from the main branch.